src/main/groovy/org/agileworks/elasticsearch/river/csv/listener/BashFileProcessorListener.groovy

```diff
@@ -100,7 +100,30 @@ class BashFileProcessorListener implements FileProcessorListener {
         try {
 
             if (file && file.exists()) {
-                def process = [file.absolutePath, args.join(" ")].execute()
+                
```

Comment:
Process running become more complex. It would be great to have a ProcessRunner interface and general implementation and extra for some OS (e.g. windows). To get the runner implementation I would follow factory pattern.

This will make testing easy and code more readable.